Module mathematica/10.0 unloaded.
Module mathematica/10.0 loaded.
Module gcc/7.1.0 unloaded.
Module gcc/7.1.0 loaded.
Module boost/1.65.1-gnu7.1-intelmpi2018 and 2 dependencies unloaded.
Module boost/1.65.1-gnu7.1-intelmpi2018 and 2 dependencies loaded.
Module eigen/3.3.3 unloaded.
Module eigen/3.3.3 loaded.
Module gsl/1.16 unloaded.
Module gsl/1.16 loaded.
(100.0, 5.0, 4000000.0, -20.0)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(119.3776641714437, 5.0, 4000000.0, -23.875532834288741)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(142.51026703029979, 5.0, 4000000.0, -28.502053406059957)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(170.12542798525891, 5.0, 4000000.0, -34.025085597051785)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(203.09176209047348, 5.0, 4000000.0, -40.618352418094695)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(242.44620170823282, 5.0, 4000000.0, -48.489240341646564)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(289.42661247167518, 5.0, 4000000.0, -57.885322494335036)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(345.51072945922181, 5.0, 4000000.0, -69.102145891844359)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(412.4626382901352, 5.0, 4000000.0, -82.492527658027043)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(492.38826317067418, 5.0, 4000000.0, -98.477652634134841)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(587.80160722749122, 5.0, 4000000.0, -117.56032144549825)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(701.7038286703829, 5.0, 4000000.0, -140.34076573407657)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(837.67764006829248, 5.0, 4000000.0, -167.53552801365851)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(1000.0, 5.0, 4000000.0, -200.0)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(1193.7766417144369, 5.0, 4000000.0, -238.75532834288737)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(1425.1026703029993, 5.0, 4000000.0, -285.02053406059986)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(1701.2542798525892, 5.0, 4000000.0, -340.25085597051782)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(2030.9176209047348, 5.0, 4000000.0, -406.18352418094696)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(2424.4620170823282, 5.0, 4000000.0, -484.89240341646564)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(2894.2661247167516, 5.0, 4000000.0, -578.85322494335037)
(3455.1072945922183, 5.0, 4000000.0, -691.02145891844361)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(4124.6263829013524, 5.0, 4000000.0, -824.92527658027052)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(4923.8826317067414, 5.0, 4000000.0, -984.77652634134824)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(5878.0160722749115, 5.0, 4000000.0, -1175.6032144549822)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:798: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_1loop > 0' failed.
(7017.0382867038297, 5.0, 4000000.0, -1403.407657340766)
(8376.7764006829238, 5.0, 4000000.0, -1675.3552801365847)
(10000.0, 5.0, 4000000.0, -2000.0)
(11937.766417144358, 5.0, 4000000.0, -2387.5532834288715)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(14251.026703029993, 5.0, 4000000.0, -2850.2053406059986)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:799: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v2_1loop > 0' failed.
(17012.54279852589, 5.0, 4000000.0, -3402.508559705178)
run_NUHMSSMNoFV.x: models/NUHMSSMNoFV/flexible_eft/THDM-II/NUHMSSMNoFV_matching_functions_1loop.cpp:436: void flexiblesusy::flexible_eft::match_1loop(const flexiblesusy::NUHMSSMNoFV_mass_eigenstates&, const flexiblesusy::THDMII_mass_eigenstates&, flexiblesusy::THDMII_mass_eigenstates&): Assertion `v1_tree > 0.0' failed.
(20309.176209047389, 5.0, 4000000.0, -4061.8352418094778)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(24244.620170823309, 5.0, 4000000.0, -4848.9240341646619)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(28942.661247167518, 5.0, 4000000.0, -5788.5322494335032)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(34551.072945922184, 5.0, 4000000.0, -6910.2145891844366)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(41246.263829013478, 5.0, 4000000.0, -8249.2527658026956)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(49238.826317067418, 5.0, 4000000.0, -9847.7652634134829)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(58780.160722749119, 5.0, 4000000.0, -11756.032144549823)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(70170.382867038366, 5.0, 4000000.0, -14034.076573407674)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(83767.764006829238, 5.0, 4000000.0, -16753.552801365848)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
(100000.0, 5.0, 4000000.0, -20000.0)
terminate called after throwing an instance of 'std::runtime_error'
  what():  Root finder did not find a solution!
